docs/visual-testing/workflows/review.md

```diff
@@ -146,6 +164,7 @@ By default, you will see diffs in a Side-by-side view, which lets you easily com
 - Zoom in or out of the snapshots.
 - Zoom to 100% or fit snapshot to the screen.
 - Highlight or hide diffs for easier discovery of the changes.
+- Open diff inspection
```

Comment:
Inspect DOM changes ?

Reply:
I think we should use unified terms across ui, docs, api and client bindings. 
- feature have many names in UI and we need to start using just one in all places. Basically it shows only dom diffs which are correlated with image diff. IMO Aplitools term `Root cause analysis` is more accurate then `Dom diff inspection` or others we are using
- option for toggling feature is called `captureDom` in client bindings and api so the same term should be used in docs and UI

![Screenshot 2024-02-23 at 10 33 51](https://github.com/saucelabs/sauce-docs/assets/119342898/14b66919-4117-4907-a002-241ea36cdf37)